### PR TITLE
Remove async/await example

### PR DESCRIPTION
**Pull request by jyoo980**
@rtholmes 

Students have been confused in lab and asking about async/await, do you think it's okay to remove this?

### CODE REVIEW

**Comment by rtholmes** [comment]
@LucasZamprogno what do you think? I'm all in favour of simplifying this and getting rid of `async` is something we do want to do.

File: project/Deliverable0.md [hunk 1]
```diff
@@ -80,20 +80,7 @@ In the meantime for the first week, you should go to a lab even if you are not r
...
-        response = await insightFacade.methodName(params);
-    } catch (err) {
-        response = err;
-    } finally {
-        expect(response).to.deep.equal(expectedValue);
-    }
-});
-```
-or you can return the promise created by your function call, and use appropriate expect statements in the `then()` and `catch()` blocks. For example:
+You can return the promise created by your function call, and use appropriate expect statements in the `then()` and `catch()` blocks. For example:
```
Comment (braxtonhall) [inline project/Deliverable0.md]: Can we start this line with `A. `? Or replace the example with with a then/catch formatted one?

Comment (jyoo980) [reply project/Deliverable0.md]: You got it champ

**Comment by rtholmes** [comment]
Lgtm

**Pull request merged**